.do/k8s-cluster/Makefile

```diff
@@ -64,21 +64,28 @@ install_nginx:
 install_certmanager:
 	helm repo add jetstack https://charts.jetstack.io
 	helm repo update
-	kubectl label namespace kube-system certmanager.k8s.io/disable-validation="true" --overwrite
-	kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v0.15.1/cert-manager.crds.yaml
-
-	# Fetch and patch issuer-letsencrypt.yaml Issuer
-	wget https://docs.codacy.com/chart/values-files/issuer-letsencrypt.yaml >/dev/null 2>&1
-	ytool -f "issuer-letsencrypt.yaml" -s metadata.namespace "kube-system" -e
-	ytool -f "issuer-letsencrypt.yaml" -s spec.acme.email "reliability@codacy.com" -e
-	kubectl apply --namespace kube-system -f issuer-letsencrypt.yaml
-
 	# Actually install cert-manager
 	wget https://docs.codacy.com/chart/values-files/values-cert-manager.yaml >/dev/null 2>&1
 	ytool -f "values-cert-manager.yaml" -s global.leaderElection.namespace "kube-system" -e
 	helm upgrade --install --version ${CERT_MANAGER_VERSION} codacy-cert-manager jetstack/cert-manager \
 			--namespace kube-system \
-			--values values-cert-manager.yaml
+			--values values-cert-manager.yaml \
+			--set installCRDs=true
+
+	kubectl label namespace kube-system certmanager.k8s.io/disable-validation="true" --overwrite
+
+	# Fetch and patch issuer-letsencrypt.yaml Issuer
+	wget https://docs.codacy.com/chart/values-files/issuer-letsencrypt.yaml >/dev/null 2>&1
+	ytool -f "issuer-letsencrypt.yaml" -s spec.acme.email "reliability@codacy.com" -e
+	ytool -f "issuer-letsencrypt.yaml" -s metadata.name "letsencrypt-codacy-dev -e
+	ytool -f "issuer-letsencrypt.yaml" -s metadata.namespace "codacy-dev -e
+	kubectl apply --namespace codacy-dev -f issuer-letsencrypt.yaml
+	ytool -f "issuer-letsencrypt.yaml" -s metadata.name "letsencrypt-codacy-sandbox -e
+	ytool -f "issuer-letsencrypt.yaml" -s metadata.namespace "codacy-sandbox -e
+	kubectl apply --namespace codacy-sandbox -f issuer-letsencrypt.yaml
+	ytool -f "issuer-letsencrypt.yaml" -s metadata.name "letsencrypt-codacy-release -e
+	ytool -f "issuer-letsencrypt.yaml" -s metadata.namespace "codacy-release -e
+	kubectl apply --namespace codacy-release -f issuer-letsencrypt.yaml
```

Comment:
We had issues with installing a single issuer shared across the 3 codacy namespaces. 
In fact, installing one issuer per namespace (environment) is a better representation of what a customer would have in their own cluster. Because of these two reasons, we are falling back to installing the 3 issuers in our cluster.